charts/testkube/values.yaml

```diff
@@ -282,6 +282,8 @@ testkube-api:
   ##ref: https://kubeshop.github.io/testkube/integrations/slack-integration/
   ## Slack token from the testkube authentication endpoint
   slackToken: ""
+  ## Slack secret to store slackToken, the key name should be SLACK_TOKEN
+  slackSecret: ""
```

Comment:
@jitapichab  Please add empty slack secret to other 3 values files together with empty slack token

Reply:
@vsukhin ready !!